ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+#### Problem
+
+...
+
+#### Steps to replicate
+
+Provide example config and message
+
+#### Expected Behavior or What you need to ask
+
+...
+
+#### Using Fluentd and out_http plugin versions
+
+* OS version
+* Fluentd v0.12 or v0.14/v1.0
+  * paste result of ``fluentd --version`` or ``td-agent --version``
```

Comment:
Why are you wrapping inline code blocks in double-backticks? It's sufficient to use a single backtick in markdown.

Reply:
This is not my change but I think it's there because some git configs actually interprete stuff inside single back-ticks and replace the text inside it by the command when writing the commit message.

Reply:
It looks like [this repository](https://github.com/stevemao/github-issue-templates) was set up specifically to show examples for issue and PR templates. If you look through the files on there (e.g., [this one](https://github.com/devspace/awesome-github-templates/blob/master/pull_request_template.md)), you'll see that it's using single backticks for this sort of thing.